bigtable/integration_test.go

```diff
@@ -80,14 +83,15 @@ var instanceToCreate string
 
 func init() {
 	if runCreateInstanceTests {
-		instanceToCreate = fmt.Sprintf("create-%d", time.Now().Unix())
+		instanceToCreate = fmt.Sprintf("bt-it-%d", time.Now().Unix())
 	}
 }
 
 func TestMain(m *testing.M) {
 	flag.Parse()
 
-	c := integrationConfig
+	env, _ := NewIntegrationEnv()
```

Comment:
You should receive the error from `NewIntegrationEnv` and fail on it if it's not nil.

Reply:
I have altered this to panic if this happens with an error.